stories/assembly-aap-gcp-intro.adoc

```diff
@@ -1,26 +1,26 @@
 ifdef::context[:parent-context: {context}]
 
 [id="assembly-intro-to-aap-on-gcp"]
-= Introduction to {PlatformNameShort} frm {GCPMarket}
+= Introduction to {AAPonGCP}
 
 :context: aap-gcp-intro
 
-{PlatformName} from {GCPMarket} is an offering that you can deploy from the GCP Marketplace portal. 
-{PlatformNameShort} from {GCPMarket} provides access to a library of Ansible content collections, and it is integrated with key GCP services, so you can start automating the deployment, configuration, and management of infrastructure and applications quickly.
+{AAPonGCP} is an offering that you can deploy from the link:https://console.cloud.google.com/marketplace[GCP Marketplace] portal. 
+{AAPonGCP} provides access to a library of Ansible content collections, and is integrated with key GCP services, so you can start automating the deployment, configuration, and management of infrastructure and applications quickly.
 
-The following {PlatformName} components are available on {PlatformNameShort} from {GCPMarket}:
+The following {PlatformName} components are available on {AAPonGCP}:
 
-* {ControllerNameStart}
-* {HubNameMain}
-* {PrivateHubNameStart}
-* {CatalogNameStart}
-* Ansible Content Collections
-* {ExecEnvNameStart}
-* Ansible content tools, including access to {InsightsName}
+* link:https://www.ansible.com/products/controller[{ControllerNameStart}]
+* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.2/html/getting_started_with_automation_hub/pr01[{HubNameMain}]
+* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html-single/managing_containers_in_private_automation_hub/index[{PrivateHubNameStart}]
+* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.2/html/getting_started_with_automation_services_catalog/index[{CatalogNameStart}]
+* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.2/html-single/publishing_proprietary_content_collections_in_automation_hub/index[Ansible Content Collections]
+* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.2/html-single/creating_and_consuming_execution_environments/index[{ExecEnvNameStart}]
+* link:https://www.ansible.com/products/automation-platform[Ansible content tools, including access to {InsightsName}]
 
 [NOTE]
 =====
-{AutomationMeshStart} is not available on {PlatformNameShort} from {GCPMarket} at this time and is planned for a later release.
+{AutomationMeshStart} is not available on {AAPonGCP} at this time and is planned for a later release.
```

Comment:
```suggestion
{AutomationMeshStart} is not available on {AAPonGCP} at this time.
```
Suggestion: don't mention the future plans, or customers will plague support, asking when it'll be included.